requirements/base.txt

```diff
@@ -1,4 +1,5 @@
 django
 regex
 future
-six
```

Comment:
I'm confused by this diff... also any idea why the tests were passing without this?